go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -213,9 +228,11 @@ func (bh *S3BackupHandle) handleAddFile(ctx context.Context, filename string, pa
 	}()
 }
 
-func calculateUploadPartSize(filesize int64) int64 {
+// this is a helper to calculate the part size, taking into consideration the minimum part size
```

Comment:
Nit, but the function comment should start with the function name. Some linters will complain about this.